docs/dev-setup.rst

```diff
@@ -90,7 +96,21 @@ To setup your local environment you should create a virtualenv and install the
 necessary requirements::
 
     mkvirtualenv --python=/usr/bin/python3.4 serviceinfo
```

Comment:
Can you maybe add something in, for the sake of us Mac users, about how to use pythonz to accomplish the same thing? pythonz is part of Calvin's standard dev environment setup for our ilk (http://caktus.github.io/developer-documentation/dev-setup.html#mac-os-x).

What this looks like:

    mkvirtualenv -p $(pythonz locate 3.4.3) serviceinfo